package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "jquery.textarea-helper",
```

Comment:
can we just call it `textarea-helper`? seems to be available: https://www.npmjs.com/package/textarea-helper